backend/src/main/java/de/nordakademie/iaa/noodle/dao/SurveyRepository.java

```diff
@@ -42,10 +43,12 @@
            SELECT DISTINCT
                survey.id AS id,
                survey.title AS title,
-                   (SELECT COUNT(response) FROM Response response WHERE response.participation.survey = survey)
+               surveySelectedTimeslot AS selectedTimeslot,
+               (SELECT COUNT(response) FROM Response response WHERE response.participation.survey = survey)
                    AS responseCount
            FROM Survey survey
            LEFT OUTER JOIN Participation participation ON participation.survey = survey
+           LEFT OUTER JOIN Timeslot surveySelectedTimeslot ON survey.selectedTimeslot = surveySelectedTimeslot
```

Comment:
Food for thought: Do we actually need those two joins, or will jpql handle it for us?

Reply:
Not really, because we need outer joins which cannot be created implicitly as far as I know.